packages/salesforcedx-vscode-core/src/orgBrowser/metadataOutlineProvider.ts

```diff
@@ -1,11 +1,11 @@
-import * as vscode from 'vscode';
-
 /*
  * Copyright (c) 2019, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+import * as vscode from 'vscode';
```

Comment:
Wow, wacky, I don't know how those imports got moved before the copyright header.  Good find.